Documentation/ApiOverview/Configuration/Index.rst

```diff
@@ -35,25 +35,13 @@ you will be directed to another manual or to a different section in this manual.
    ConfigurationSyntax
    ConfigurationMethods
    ../TypoScriptSyntax/Index
-   ../Yaml/Index
```

Comment:
Changes like these are out of scope

Reply:
A separate commit should be enough, not?